app/javascript/gobierto_data/webapp/components/sets/data/SQLEditorCode.vue

```diff
@@ -72,6 +88,18 @@ export default {
       if (newValue !== oldValue) {
         this.setEditorValue(newValue);
       }
+    },
+    resetQueryDefault(newValue, oldValue) {
+      if (newValue !== oldValue) {
+        this.setEditorValue(this.queryDefault);
+        this.editor.focus()
+      }
+    },
+    revertQuerySaved(newValue, oldValue) {
+      if (newValue !== oldValue) {
+        this.setEditorValue(this.queryRevert);
+        this.editor.focus()
+      }
```

Comment:
Try to simplify this logic, printing only the queryStored for this component. This changes are populated upstream.